saemref/defaults.yaml

```diff
@@ -6,6 +6,16 @@ saemref:
     port:
     anonymous_user:
     anonymous_password:
+    cookie_secret_key:
+    {% if grains['os_family'] == 'RedHat' and grains['osmajorrelease'] == '6' %}
+    wsgi: false
+    pool_size: 4
```

Comment:
`pool_size` could have the same default value even for centos6 (as do `wsgi_threads` and `wsgi_workers`)